crates/release_plz_core/src/cargo.rs

```diff
@@ -68,6 +79,10 @@ pub fn is_published(index: &mut Index, package: &Package) -> anyhow::Result<bool
     Ok(is_in_cache(index, package))
 }
 
+pub async fn is_published_sparse(index: &SparseIndex, package: &Package) -> anyhow::Result<bool> {
+    is_in_cache_sparse(index, package).await
+}
```

Comment:
This function only calls another function. Should we remove it? For example, we can directly call `is_in_cache_sparse`